gnovm/tests/imports.go

```diff
@@ -138,7 +138,7 @@ func TestStore(rootDir, filesPath string, stdin io.Reader, stdout, stderr io.Wri
 				pkg.DefineGoNativeType(reflect.TypeOf((*fmt.Formatter)(nil)).Elem())
 				pkg.DefineGoNativeValue("Println", func(a ...interface{}) (n int, err error) {
 					// NOTE: uncomment to debug long running tests
-					fmt.Println(a...)
+					// fmt.Println(a...)
```

Comment:
:)

Reply:
(resolve this comment, just thought it was funny)

Reply:
:rofl: